src/assets/styles/base.css

```diff
@@ -0,0 +1,155 @@
+:root {
+  --root-font-size: 16px;
+
+  --global-layout-size: 80rem; /* 1280px */
```

Comment:
I like to namespace my variables like such and then have the option to overwrite them at the component level. You can also declare variables inside of a CSS class/selector and have it be scoped to it and its children.